Masquerade4997Library/src/main/java/Library4997/MasqControlSystems/MasqPID/MasqAnglePIDController.java

```diff
@@ -0,0 +1,107 @@
+package Library4997.MasqControlSystems.MasqPID;
```

Comment:
I might be trippin, but this entire class looks the same as a normal PIDController.